multiwallet.go

```diff
@@ -104,6 +107,8 @@ func NewMultiWallet(rootDir, dbDriver, netType, politeiaHost string) (*MultiWall
 		txAndBlockNotificationListeners:  make(map[string]TxAndBlockNotificationListener),
 		accountMixerNotificationListener: make(map[string]AccountMixerNotificationListener),
 	}
+	// API service.
+	mw.ApiService = api.NewService(chainParams)
```

Comment:
It's not immediately clear to me what `ApiService` mean. Can we find a more descriptive name for the package? `API` is a generic term, so is `Service`.